WcaOnRails/app/models/competition.rb

```diff
@@ -593,6 +610,10 @@ def country
     Country.find_by_id(countryId)
   end
 
+  def continent
+    country ? Continent.find_by_id(country.continentId) : nil
```

Comment:
Hmm I think so, but that behavior is kind of different, right? If `country` is not defined, it feels pretty silly/kind of scary to do a database lookup knowing we don't expect to find anything.